src/main/java/com/xebialabs/overthere/gcp/GcpMetadataKeyManager.java

```diff
@@ -50,6 +53,7 @@
     private ProjectCredentials projectCredentials;
     private GcpSshKey gcpSshKey;
     private Compute computeService;
+    private static final List<String> SCOPES = Arrays.asList(ComputeScopes.CLOUD_PLATFORM);
```

Comment:
It seems that this constant is not used anywhere can you remove it?

Reply:
Sure :) 👍